client/my-sites/checklist/onboardingChecklist.js

```diff
@@ -26,27 +27,31 @@ const tasks = {
 		duration: '2 mins',
 		completedTitle: 'You uploaded an avatar',
 		completedButtonText: 'Change',
+		url: '/me',
 	},
 	blogname_set: {
 		title: 'Personalize your site',
 		description: 'Give your site a descriptive name to entice visitors.',
 		duration: '1 min',
 		completedTitle: 'You updated your site title',
 		completedButtonText: 'Edit',
+		url: '/settings/general/$siteSlug',
```

Comment:
This is ok for now but when we have the trampolines, this should go to the Site Identity section in the Customizer.